src/components/entries/NumberField.js

```diff
@@ -0,0 +1,109 @@
+import {
+  useMemo
+} from 'preact/hooks';
+
+function NumberField(props) {
+
+  const {
+    debounce,
+    disabled,
+    id,
+    label,
+    max,
+    min,
+    onInput,
+    step,
+    value = ''
+  } = props;
+
+  const handleInput = useMemo(() => {
+    return debounce(event => {
+
+      const {
+        validity,
+        value
+      } = event.target;
+
+      if (validity.valid) {
+        onInput(value ? parseInt(value, 10) : undefined);
```

Comment:
By for example setting `step` to something like `0.01` the user would also be able to insert floats into the number field (e.g., in the example `10.12` would be a valid input). Still the parseInt will cut off the decimal places (so in the example `10` would be passed to `onInput`). Any reason we do this? Alternatively we could use `parseFloat` right?

Reply:
Good point, I didn't think about it.

If we use `parseFloat`, would that lead to always commit floating point numbers? Let's try it out.

![image](https://user-images.githubusercontent.com/9433996/125445585-53aa9c6d-f7b3-4ad3-9d36-ec05ae90caa3.png)

Reply:
Seems like we're good to go with this, also ensuring we can have floating-point numbers via `step`.

Reply:
Fixed via https://github.com/bpmn-io/properties-panel/pull/76/commits/b81eedbfc88be75b7c9d877ac217bc9c36a7d784